x-pack/plugins/data_enhanced/server/search/es_search_strategy.ts

```diff
@@ -81,7 +81,7 @@ async function asyncSearch(
   const path = encodeURI(request.id ? `/_async_search/${request.id}` : `/${index}/_async_search`);
 
   // Wait up to 1s for the response to return
-  const query = toSnakeCase({ waitForCompletionTimeout: '100ms', ...queryParams });
+  const query = toSnakeCase({ waitForCompletionTimeout: '1ms', keepAlive: '1m', ...queryParams });
```

Comment:
Why did you end up lowering it back to 1ms?

Reply:
Oops, that's an accident. I was just testing with the lower amount.